### PR TITLE
Fixes #32

### LINKED ISSUE #32: Reference LICENSE into README

### PR DESCRIPTION
**Pull request by tarlepp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -254,3 +254,8 @@ $client = new Snorlax\RestClient([
 
 # Contributing
 Please see the [CONTRIBUTING.md](.github/CONTRIBUTING.md) file for guidelines.
+
+# LICENSE
+[The MIT License (MIT)](LICENSE)
+
+Copyright (c) 2016 Pedro Maia
```
Comment (pedrommone) [inline README.md]: Can you rename to `EzDelivery`?

This is not my personal project :)

Comment (tarlepp) [inline README.md]: That was the text in LICENSE file

Comment (tarlepp) [inline README.md]: Also created #35 to fix that LICENSE file.

Comment (pedrommone) [inline README.md]: Can I request you to change it in LICENSE file as well?

Comment (tarlepp) [inline README.md]: See previous comment and #35

**Comment by pedrommone** [comment]
@tarlepp sorry but @nitishkd made the PR before you. I've approved your PR #35. If you think you can improve his PR feel free to help! :)

**Comment by tarlepp** [comment]
Actually I was the first, but I'll improve that one.

**Pull request closed without merging**